experiment/runner.py

```diff
@@ -466,6 +469,8 @@ def main():
             'trial_id': str(environment.get('TRIAL_ID')),
         })
     experiment_main()
+    if fuzzer_errored_out:
```

Comment:
nit: return int(fuzzer_errored_out)

Reply:
Disagree on this. The current version seems clearer and more explicit.
It seems weird to convert `True` to a returncode.